apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2015,23 +2059,52 @@ check_withdraw_error_msg(Msg, D) ->
 check_update_err_msg(Msg, D) ->
     check_op_error_msg(?UPDATE_ERR, Msg, D).
 
+check_shutdown_err_msg(#{ channel_id := ChanId } = Msg,
+                       #data{ on_chain_id = ChanId
+                            , op = Op } = D) ->
+    case Op of
+        #op_close{data = #op_data{signed_tx = SignedTx}} ->
+            TxHash = aetx_sign:hash(SignedTx),
+            case aec_chain:find_tx_location(TxHash) of
+                Block when is_binary(Block) ->
+                    %% Tx is already on chain - ignore error msg
+                    {error, tx_already_on_chain};
```

Comment:
Debug logs for these 2 cases would be nice as this could help troubleshoot issue and/or usage patterns later.